src/db/migrations/migration1.js

```diff
@@ -0,0 +1,31 @@
+const _ = require('lodash');
+const fs = require('fs-extra');
+const { getLogger } = require('../../utils/logger');
+const Utils = require('../../utils');
+
+async function addLanguageField(db) {
+  try {
+    await db.update({ language: { $exists: false } }, { $set: { language: 'en-US' } }, { multi: true });
+  } catch (err) {
+    throw Error(`DB update Error: ${err.message}`);
+  }
+}
+
+
+async function migration1(db) {
+  const migrationTrackPath = `${__dirname}/../migrations.dat`;
+  try {
+    let lastMigrate = Number(await fs.readFileSync(migrationTrackPath).toString().split('=')[1].trim());
```

Comment:
lets keep track of the `lastMigrate` number in `applyMigrations()` and pass it in as a param to the migration: `migration(db, lastMigrationNum)`